tests/e2e/tests/test_redirects.py

```diff
@@ -150,20 +150,23 @@ def _verify_winxp_redirect_rules(self, base_url, product_alias, user_agent,):
         elif 'esr' in product_alias:
             assert '38.5.1esr.exe' in parsed_url.path
         elif product_alias in ['49.0b8', '49.0b10', '49.0b37']:
-            # beta versions are pinned to 49.0b10
-            assert '49.0b10.exe' in parsed_url.path
+            # beta versions are now a moving target, check that it is never
+            # less than 49
+            assert 49 <= self._extract_version_num(parsed_url.path)
         elif product_alias in ['firefox-sha1']:
             assert '49.0b8.exe' in parsed_url.path
         elif product_alias in ['beta-latest']:
             # beta-latest is a moving target, check that it is never
             # less than 49
-            extracted_ver_num = parsed_url.path.split('Firefox%20Setup%20')[1].split('.')[0]
-            assert 49 <= int(extracted_ver_num)
+            assert 49 <= self._extract_version_num(parsed_url.path)
```

Comment:
As above, the comment says 'never less than 49' but the assert is less than or equal to.